lib/event_store_client/adapters/grpc/shared/streams/process_response.rb

```diff
@@ -18,7 +19,7 @@ def call(response, skip_deserialization, skip_decryption)
             return unless response.event&.event
 
             Success(
-              EventDeserializer.new.call(response.event.event, skip_decryption: skip_decryption)
+              config.mapper.deserialize(response.event.event, skip_decryption: skip_decryption)
```

Comment:
What in case somehow `config.mapper` or not set at all maybe it would be great to check if the mapper responds to the `deserialize`
Or we can assure/validate it in the config?

Reply:
Well, the default value of it is set to `EventStoreClient::Mapper::Default.new`, and documentation says that there are two possible values for `config.mapper` option. Not sure if we should guard users from setting it to `nil`. I mean, if ruby dev would decide to bypass that potential restriction - he could easily do that. In worst case the app will throw `NoMethodError`.

Reply:
Ok good.